sites/microservices.conf

```diff
@@ -21,12 +21,12 @@ server {
         www.zooteach.org
     ;
 
-    set $swarm_uri "https://microservices";
+    set $cluster_uri "https://microservices";
```

Comment:
does this URL resolve without a domain name extension?

Reply:
Yup, it resolves to the `upstream` defined at the top of the file.